README.md

```diff
@@ -130,6 +130,31 @@ To dump VCDs (into the test_run_dir subfolder) using sbt:
 testOnly chiseltest.tests.BasicTest -- -DwriteVcd=1
 ```
 
+## Verilator Backend
+
+To run your tests using the [verilator](https://www.veripool.org/wiki/verilator) simulator:
+
+1. Ensure that you have a _supported version_ of verilator installed.
+   You can [consult the CI script](https://github.com/ucb-bar/chisel-testers2/blob/master/.github/workflows/test.yml#L38)
+   to find out which versions you can use.
+2. Enable the Verilator Backend by adding the right annotation to your tests:
+
+```.scala
+// [...]
+import chiseltest.experimental.TestOptionBuilder._
+import chiseltest.internal.VerilatorBackendAnnotation
```

Comment:
Can this be moved (or duplicated) to experimental, so people aren't importing internal items?

Reply:
I would be happy to. I am not sure what the story around the Verilator backend is supposed to be.
To be honest, I feel like `withAnnotation`, `VerilatorBackendAnnotation` and `TreadleBackendAnnotation` should just be available when you `import chiseltest._`. What do you think?

Reply:
Is `withAnnotation` experimental?
But I'm happy with `VerilatorBackendAnnotation` and friends coming with `withAnnotation`.

The story around the Verilator backend is that it lives in legacy because it's largely an imported codebase, that hasn't been reviewed for code quality or style. But as `VerilatorBackendAnnotation` is more an interface class that doesn't have much in the way of implementation, I think that can be closer to mainline.

Not sure why `VerilatorBackendAnnotation` is internal - maybe we forgot about it? Or was there another public-facing API intended for making tests run on Verilator? @chick ?

Reply:
I think I am going to remove the example from this PR for now. Changing the package of important annotations should be a PR on its own.